### DISCUSSION TITLE
Kobo Libra H2O Battery Level Check

### DISCUSSION THREAD

**Discussion by dcruzbosco** [koreader/koreader#11445] [Q&A] [answered]
Hi,

Is there any command I can run in KOReader terminal to check actual battery level?

Device shows good amount of battery, in both Nickel and KOReader, but randomly shuts down when reading, or trying to update, or even starting back up.
I'm guessing battery is at little more than zero, which is why it works for a while, but is reporting more.

Thanks for your help!

**Comment by Frenzie** [comment, marked as answer]
You could take a look around in `/sys/class/power_supply/battery` but in principle those'll all just be a few different angles to approach the same thing. `/sys/class/power_supply/battery/capacity` should be the same value that's read out by Nickel and KOReader, but in that directory you can also get more directly at some of the things that number is made up of. I wouldn't expect them to help any though, otherwise the value in `capacity` would be more helpful in the first place.

**Reply by dcruzbosco** [reply]
Thanks @Frenzie! Yes, `capacity` shows the same as reported by Nickel and KOReader.
I'm going to try draining the battery and recharging to see if it fixes it.
Else will reset the device.
